src/sql/workbench/services/notebook/browser/notebookServiceImpl.ts

```diff
@@ -606,4 +606,22 @@ export class NotebookService extends Disposable implements INotebookService {
 			editor.navigateToSection(sectionId);
 		}
 	}
+
+	/**
+	 * Trusts a notebook with the specified URI.
+	 * @param notebookUri The notebook URI to set the trusted mode for.
+	 * @param isTrusted True if the notebook is to be trusted, false otherwise.
+	 */
+	async setTrusted(notebookUri: URI, isTrusted: boolean): Promise<boolean> {
+		if (isTrusted) {
```

Comment:
I'd rather we get rid of all untrusted logic if we're not going to support that. You mentioned that you need it for testing - could you point out where that is? There should be some other way to achieve that functionality (such as clearing the config or something) without needing to use this.

Reply:
# resolved